code/package.json

```diff
@@ -335,7 +336,7 @@
     "ts-jest": "^26.4.4",
     "ts-node": "^10.4.0",
     "tsup": "^6.2.2",
-    "typescript": "4.7.4",
+    "typescript": "~4.6.3",
```

Comment:
downgrade?

Reply:
More likely ensuring we use the same version everywhere.